.github/workflows/nodejs.yml

```diff
@@ -18,9 +18,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: actions/setup-node@v2.4.0
+      - uses: actions/setup-node@v2
```

Comment:
dependabot is what changed from just specifying major: #620. But if renovate leaves it alone, I'm happy. I was unable to find any docs specifying the behavior - do you know?